config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/cloud-bulldozer/benchmark-operator
+  newTag: latest-amd64
```

Comment:
I think you uploaded this file by mistake, (It gets modified whenever you make deploy)

Reply:
fixed